src/components/dialog/dialog.config.js

```diff
@@ -7,5 +7,7 @@ export const DIALOG_SIZES = [
   "large",
   "extra-large",
 ];
-
 export const TOP_MARGIN = 20;
+export const CONTENT_TOP_PADDING = 24;
```

Comment:
This was previously 20px if you look at `DialogInnerContentStyle`

Reply:
It's a requirement of this ticket to update that value to 24

Reply:
Would be good to put all the intended changes in the PR description 👍🏻